src/content/configuration/externals.md

```diff
@@ -321,6 +321,7 @@ Supported types:
 - `'system'`
 - `'promise'` - same as `'var'` but awaits the result (async module)
 - `'import'` - uses `import()` to load a native EcmaScript module (async module)
+- `'script'` - load script with HTML `<script>` element and use the global variable defined.
```

Comment:
Is this more accessible than `global variable defined`? I think it assumes you understand context well enough
```suggestion
- `'script'` - load script with HTML `<script>` element and store the result in a selected global variable
```

Reply:
How about?:

> load script exposing predefined global variables with HTML `<script>` element

From what I learned so far, we can only load external scripts here with `<script>` element when they expose predefined global variables like jquery did with global `window.jQuery` and `window.$`, otherwise it'll fail. Also we can't use random variable name but those predefined ones in `global` placeholder.

Reply:
Yeah reads much better and also makes sense, lets get Tobias to comment if this is right tho